csm/core/controllers/information.py

```diff
@@ -49,7 +49,7 @@ async def post(self):
         resource = self.request.match_info[const.ARG_RESOURCE]
         # Check for valid Resource
         if resource not in const.VERSION_RESOURCES:
-            raise CsmNotFoundError(f"{resource} is not valid")
+            raise CsmNotFoundError(f"Invalid resource: {resource}")
```

Comment:
raise InvalidRequest error as discussed

Reply:
Updated

Reply:
This code is part of other API. updated error as per suggetion